app/core/api_errors.py

```
@@ -0,0 +1,445 @@
+
+            start_message["headers"] = replace_headers(
+                start_message.get("headers", []),
+                body,
+                content_type=content_type if changed else None,
+            )
+            await send(start_message)
+            await send({"type": "http.response.body", "body": body})
+
+        try:
+            await self.app(scope, receive, send_sanitized)
+        except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)